test/extensions/filters/network/thrift_proxy/integration_test.cc

```diff
@@ -119,6 +119,42 @@ class ThriftConnManagerIntegrationTest
       }
     });
 
+    std::tie(std::ignore, std::ignore, std::ignore, payload_passthrough_) = GetParam();
+
+    if (payload_passthrough_) {
```

Comment:
Suggest factoring this out to a different method or test.

Reply:
I need to add `payload_passthrough=true` to all existed tests, so here is another test param. 
Will it be better if I replace this part with `tryInitializePassthrough()` ?

Reply:
This was just a code structure suggestion. If you do the below suggested change, this probably doesn't matter.